File: app/src/main/java/acr/browser/lightning/utils/Preconditions.java
package acr.browser.lightning.utils;

import androidx.annotation.Nullable;

public final class Preconditions {

    private Preconditions() {}

    /**
     * Ensure that an object is not null
     * and throw a RuntimeException if it
     * is null.
     *
     * @param object check nullness on this object.
     */
    public static void checkNonNull(@Nullable Object object) {
        if (object == null) {
            throw new RuntimeException("Object must not be null");
        }
    }
}

File: app/src/main/java/acr/browser/lightning/utils/Preconditions.kt
package acr.browser.lightning.utils;

object Preconditions {

    /**
     * Ensure that an object is not null
     * and throw a RuntimeException if it
     * is null.
     *
     * @param obj check nullness on this object.
     */
    fun checkNonNull(obj: Any?) {
        if (obj == null) {
            throw RuntimeException("Object must not be null")
        }
    }
}
